### ISSUE TITLE
Attempting to add in gradle maven dependency, asm mixin throwables error

### ISSUE THREAD

**Issue by Lothrazar** [architectury/architectury#36]
Mixin apply failed architectury.mixins.json

loaded mods
mc_version=1.16.5
forge_version=36.0.12
mappings_version=20201028-1.16.3
jei_version=jei-1.16.4:7.6.1.65
curios_version=1.16.4-4.0.3.5
patchouli_version=1.16.4-48
botania_version=1.16.4-410
cyclic-1.16.5-1.1.5

in my gradle file i am putting the cursemaven for the mod latest release version

for https://www.curseforge.com/minecraft/mc-mods/architectury-forge/files/3188530    

as a 
dependencies {
    implementation fg.deobf("curse.maven:architectury-forge-419699:3188530")
// matches with https://www.curseforge.com/minecraft/mc-mods/architectury-forge/files/3188530    

    
    }
repositories {
  maven { url 'https://www.cursemaven.com'  }
}

            property 'mixin.env.disableRefMap', 'true'
            

error is the following.

```
[13:10:26] [pool-3-thread-1/FATAL] [mixin/]: Mixin apply failed architectury.mixins.json:MobSpawnSettingsBuilderAccessor -> net.minecraft.world.biome.MobSpawnInfo$Builder: org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException No candidates were found matching mobSpawnCosts:Ljava/util/Map; in net/minecraft/world/biome/MobSpawnInfo$Builder for architectury.mixins.json:MobSpawnSettingsBuilderAccessor->@Accessor[FIELD_GETTER]::getMobSpawnCosts()Ljava/util/Map; [INJECT Applicator Phase -> architectury.mixins.json:MobSpawnSettingsBuilderAccessor -> Apply Accessors ->  -> Locate -> architectury.mixins.json:MobSpawnSettingsBuilderAccessor->@Accessor[FIELD_GETTER]::getMobSpawnCosts()Ljava/util/Map;]
org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException: No candidates were found matching mobSpawnCosts:Ljava/util/Map; in net/minecraft/world/biome/MobSpawnInfo$Builder for architectury.mixins.json:MobSpawnSettingsBuilderAccessor->@Accessor[FIELD_GETTER]::getMobSpawnCosts()Ljava/util/Map; [INJECT Applicator Phase -> architectury.mixins.json:MobSpawnSettingsBuilderAccessor -> Apply Accessors ->  -> Locate -> architectury.mixins.json:MobSpawnSettingsBuilderAccessor->@Accessor[FIELD_GETTER]::getMobSpawnCosts()Ljava/util/Map;]
	at org.spongepowered.asm.mixin.gen.AccessorInfo.findTarget(AccessorInfo.java:518) ~[mixin-0.8.2.jar:0.8.2+unknown-b0.git-unknown] {}
```

**Comment by MaxNeedsSnacks** [comment]
i‘m not 100% certain, but i remember something about fg breaking with different mappings across projects?

i‘ll try to investigate what’s going on here

**Comment by shedaniel** [comment]
Why are you disabling the refmap? The refmap is how it remaps, also I don't think ForgeGradle properly handles remaps anyways.

**Comment by Lothrazar** [comment]
i thought you had to disable it. for mixin mods. ill try taking it out

Really im trying to get kubejs set up thro maven. 

Maybe theres a better one than curse maven im not sure

    
         implementation fg.deobf("curse.maven:architectury-forge-419699:3188530")
	  implementation fg.deobf("curse.maven:rhino-416294:3187177")
	  implementation fg.deobf("curse.maven:kubejs-238086:3137415")

**Comment by shedaniel** [comment]
Mixin mods still will not work probably under ForgeGradle.

**Comment by MaxNeedsSnacks** [comment]
![image](https://user-images.githubusercontent.com/12750501/111339879-6b571a00-8678-11eb-8d16-acf4c3e2105a.png)

TL;DR: this is an external issue with Mixingradle that can be temporarily fixed by doing the things detailed in https://github.com/SpongePowered/Mixin/issues/462#issuecomment-791370319

Let's hope a more permanent fix in implemented in FG before 4.1's API is frozen
